tests/lib/nested.sh

```diff
@@ -951,7 +951,7 @@ nested_add_file_to_vm() {
     ubuntuSeedDev="${devloop}p2"
 
     # Wait for the partition to show up
-    retry -n 2 --wait test -b "${ubuntuSeedDev}" || true
+    retry -n 2 --wait 1 test -b "${ubuntuSeedDev}" || true
```

Comment:
should we remove the || true? I mean, this could hide errors like this one.

Reply:
No, because if it is fails, we have a fall back right after.

Reply:
So in theory, it can fail. This is because losetup has a race bug which fix is not released yet.